bananas_cli/cli.py

```diff
@@ -43,6 +43,11 @@ async def cli(ctx, api_url, tus_url, client_id):
     ctx.obj = session
 
     await session.start()
+
+    os_args = click.get_os_args()
+    if "-h" in os_args or "--help" in os_args:
+        return
+
```

Comment:
I get what you want to do, but boy, this feels ugly :D No clue if it can be done nicer .. need to think about it :)

Reply:
The main issue is that when using click we have a two-staged approach: 
cli.py -> Authenticates
commandxyz.py -> Executes the command

If you specify --help, it will apply to the command, so authentication is done first, then help for the command is shown. An alternative solution would be to get rid of the first stage and do authentication in the command stage. It would require a lot more re-architecting, but would not require this weirdness, I think.